api/src/main/java/org/openmrs/module/mirebalais/require/RequireUtil.java

```diff
@@ -71,6 +71,10 @@ public static String patientYoungerThan(int age) {
         return "patient.person.age <= " + age;
     }
 
+    public static String patientYoungerThanMonths(int ageMonths) {
```

Comment:
I have no idea why whoever implemented patientYoungerThan(int) decided to make that actually return "younger than or equal to", but let's not make that mistake again here.  If it's youngerThan, then use < ageMonths.  If it's youngerThanOrEqualTo, then name the method that.

I'd also be more explicit with the name like "patientYoungerThanMonthsAtVisitStart", so it's clear that this is relative to the visit start date, rather than the current date, especially since the other "patientYoungerThan" method doesn't take the visitStartdate into play at all.

Speaking of which...we should probably also change the patientYoungerThan method to factor in the visit start date, and ideally change it so that the method matches the behavior as discussed above.  @mogoodrich ?

Reply:
We should fix the name of the patientYoungerThan method, and, yes, it should likely factor in the visit start date. (we can check to see where it is actually used, but it seems likely we want age at visit, not current age)

Reply:
Fixed. I'm not sure if `AtVisitStart` is really useful, since I can't think of anything in the visit context that *isn't* relative to the visit/encounter time. The actual current date is almost never relevant.

Reply:
I won't push on the "AtVisitStart" - I don't really care that much.  But there could definitely be other dates of interest.  the current date _might_ be what is actually desired, for an in-patient stay longer than a day, or we might have use cases in the future where we want to have logic based around "ageAtProgramEnrollment" or something.  So this helps to be clear.  But I don't care enough to worry for now.